src/index.js

```diff
@@ -8,23 +8,26 @@ import { BrowserRouter } from "react-router-dom";
 import { ProductProvider } from "./Context/ProductContext";
 import { FilterProvider } from "./Context/FilterContext";
 import { CartProvider } from "./Context/CartContext";
+import { WishProvider } from "./Context/WishListContext";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
-    <CartProvider>
-      <FilterProvider>
-        <ProductProvider>
-          <CategoryProvider>
-            <BrowserRouter>
-              <App />
-            </BrowserRouter>
-          </CategoryProvider>
-        </ProductProvider>
-      </FilterProvider>
-    </CartProvider>
+    <WishProvider>
```

Comment:
Just a suggestion make one index file for all context providers and wrap all in one then import in index.css

Reply:
+1

Reply:
Ok... Will do that while refactoring code for sure. thanks for the suggestion.